src/sentry/seer/code_review/webhooks/task.py

```diff
@@ -35,7 +39,38 @@ def _call_seer_request(
     """
     assert github_event != GithubWebhookType.CHECK_RUN
     # XXX: Add checking options to prevent sending events to Seer by mistake.
-    make_seer_request(path=SeerEndpoint.SENTRY_REQUEST.value, payload=event_payload)
+    make_seer_request(path=SeerEndpoint.OVERWATCH_REQUEST.value, payload=event_payload)
```

Comment:
We're going to start calling the overwatch endpoint unless we believe it would be beneficial to start afresh.